compiler/rustc_driver_impl/src/lib.rs

```diff
@@ -59,7 +59,7 @@ use rustc_session::getopts::{self, Matches};
 use rustc_session::lint::{Lint, LintId};
 use rustc_session::output::collect_crate_types;
 use rustc_session::{EarlyDiagCtxt, Session, config, filesearch};
-use rustc_span::FileName;
+use rustc_span::{FileName, def_id::LOCAL_CRATE};
```

Comment:
nit:

```suggestion
use rustc_span::FileName;
use rustc_span::def_id::LOCAL_CRATE;
```

to be in-line with the rest of rustc_driver_impl.